cargo-paradedb/src/fixtures/mod.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2023-2024 Retake, Inc.
```

Comment:
I'm not convinced we need a fixtures directory for this. This feels extra complicated